uber/config.py

```diff
@@ -95,7 +95,7 @@ def PREREG_AGE_GROUP_OPTS(self):
 
     @property
     def DISPLAY_ONEDAY_BADGES(self):
```

Comment:
Minor fix unrelated to the rest of this which I noticed while making this change.  ``days_before`` isn't available yet, so we use the ``sa`` prefix which we implemented to avoid circular dependencies.